tracing.cabal

```diff
@@ -1,73 +1,73 @@
-cabal-version: 3.0
-
-name: tracing
-version: 0.0.7.2
-synopsis: Distributed tracing
+cabal-version:      3.0
+name:               tracing
+version:            0.0.7.2
+synopsis:           Distributed tracing
 description:
   An OpenTracing-compliant, simple, and extensible distributed tracing library.
 
-category: Web
-homepage: https://github.com/mtth/tracing
-license: BSD-3-Clause
-license-file: LICENSE
-author: Matthieu Monsch
-maintainer: mtth@apache.org
-copyright: 2020 Matthieu Monsch
-
-build-type: Simple
+category:           Web
+homepage:           https://github.com/mtth/tracing
+license:            BSD-3-Clause
+license-file:       LICENSE
+author:             Matthieu Monsch
+maintainer:         mtth@apache.org
+copyright:          2020 Matthieu Monsch
+build-type:         Simple
 extra-source-files: README.md
-
-tested-with: GHC == { 8.10.7, 9.0.2, 9.2.8, 9.4.8, 9.6.4, 9.8.2 }
+tested-with:
+  GHC ==8.10.7 || ==9.0.2 || ==9.2.8 || ==9.4.8 || ==9.6.4 || ==9.8.2
 
 source-repository head
-  type: git
+  type:     git
   location: https://github.com/scrive/tracing
 
 library
-  hs-source-dirs: src
+  hs-source-dirs:   src
   exposed-modules:
-      Control.Monad.Trace
-    , Control.Monad.Trace.Class
-    , Monitor.Tracing
-    , Monitor.Tracing.Local
-    , Monitor.Tracing.Zipkin
-  other-modules:
-      Control.Monad.Trace.Internal
+    Control.Monad.Trace
+    Control.Monad.Trace.Class
+    Monitor.Tracing
+    Monitor.Tracing.Local
+    Monitor.Tracing.Zipkin
+
+  other-modules:    Control.Monad.Trace.Internal
   build-depends:
-      aeson >= 2.0
-    , base >= 4.14 && < 5
-    , base16-bytestring >= 1.0
-    , bytestring >= 0.10
-    , case-insensitive >= 1.2
-    , containers >= 0.6
-    , exceptions >= 0.10
-    , http-client >= 0.5
-    , lifted-base >= 0.2.3
-    , monad-control >= 1.0
-    , mtl >= 2.2
-    , network >= 2.8
-    , random >= 1.1
-    , stm >= 2.5
-    , text >= 1.2
-    , time >= 1.8
-    , transformers >= 0.5
-    , transformers-base >= 0.4
-  ghc-options: -Wall
+    , aeson              <=2.3
+    , base               <5
+    , base16-bytestring  <=1.1
+    , bytestring         <=0.13
```

Comment:
1°) I added them because the CI steps generated by haskell-ci requested them. I had originally blindly followed the CI, which suggested using `cabal gen-bounds`, but of course this broke when testing against various GHC versions.
2°) I agreed they are extremely annoying, because I had to spend quite a while fixing them. I can remove the bound check on the CI if need be (and remove bounds in the cabal file right after). Would this be a good solution ?